src/arweave_address.ts

```diff
@@ -7,15 +7,19 @@ export class ArweaveAddress {
 		}
 	}
 
-	equalsAddress(other: ArweaveAddress): boolean {
-		return this.address === other.address;
+	[Symbol.toPrimitive](hint: 'number' | 'string' | 'default'): string {
```

Comment:
Was this change necessary for this PR?

Reply:
Actually no. That's why I asked you to review that specific file :monocle_face: 
> @arielmelendez would you take a look at src/arweave_address.ts? I think that's relevant for you